plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -230,12 +209,64 @@ function extract(pluginPackagesPath, outputPlugins) {
  * @returns {0} Exit code, where 0 is success.
  */
 function start() {
+  /**
+   * Copies the built plugin to the app config folder ~/.config/Headlamp/plugins/
+   *
+   * Adds a webpack config plugin for copying the folder.
+   */
+  function copyToPluginsFolder(webpackConfig) {
+    const packageJson = JSON.parse(fs.readFileSync('package.json', 'utf8'));
+
+    // @todo: should the whole package name be used here,
+    //    and the load be fixed to use? What about namespace packages?
```

Comment:
Good point. Maybe making the namespace as part of the target path? `@kinvolk/podcounter-headlamp-plugin` -> `kinvolk-podcounter-headlamp-plugin`?

Reply:
That seems fine... I think. Or expanding the loading parts of the code so it can work with namespaces like `@kinvolk/package` properly. Need to think it through a bit more, with how that could affect namespace collisions.

It's not the topic of this PR so I've created an issue for this one: https://github.com/kinvolk/headlamp/issues/680